### ISSUE TITLE
clang_rt.profile-aarch64.lib is missing from Windows release version 16.0.0

### ISSUE THREAD

**Issue by pulidocr** [llvm/llvm-project#62810]
`clang_rt.profile-aarch64.lib` used to be included in Windows on ARM64 binaries, but now it's not. I downloaded and extracted `LLVM-15.0.6-woa64.exe` and `LLVM-16.0.0-woa64.exe` from https://github.com/llvm/llvm-project/releases and noticed it was missing starting from 16.0.0.

```
F:\scratch>dir LLVM-15.0.6-woa64\lib\clang\15.0.6\lib\windows
 Volume in drive F is New Volume
 Volume Serial Number is 6AD8-542C

 Directory of F:\scratch\LLVM-15.0.6-woa64\lib\clang\15.0.6\lib\windows

05/19/2023  10:06 AM    <DIR>          .
05/19/2023  10:06 AM    <DIR>          ..
12/06/2022  07:16 AM           497,718 clang_rt.builtins-aarch64.lib
12/06/2022  04:42 AM           372,608 clang_rt.profile-aarch64.lib
               2 File(s)        870,326 bytes
               2 Dir(s)  246,240,133,120 bytes free

F:\scratch>dir LLVM-16.0.0-woa64\lib\clang\16\lib\windows
 Volume in drive F is New Volume
 Volume Serial Number is 6AD8-542C

 Directory of F:\scratch\LLVM-16.0.0-woa64\lib\clang\16\lib\windows

05/19/2023  10:09 AM    <DIR>          .
05/19/2023  10:09 AM    <DIR>          ..
03/21/2023  10:48 PM           247,380 clang_rt.builtins-aarch64.lib
               1 File(s)        247,380 bytes
               2 Dir(s)  246,239,948,800 bytes free
```

**Comment by pulidocr** [comment]
Fixed title.

**Comment by pulidocr** [comment]
Hi @omjavaid 

I noticed you handle the Windows on ARM64 releases. Apologies in advance for the sudden ping.

Would you happen to know why this lib isn't being built starting at 16.0.0? Would it be possible to start building it for the 17 release?

**Comment by omjavaid** [comment]
Let me investigate this and get back to you.

**Comment by omjavaid** [comment]
@pulidocr I have found the issue in our downstream release script that was causing us to miss the profile lib. This will be fixed in 17.0.0 Final version.

**Comment by pulidocr** [comment]
Awesome, thank you!
